CMakeLists.txt

```diff
@@ -54,11 +58,6 @@ else()
   return()
 endif()
 
-# The cmake documentation states that these are set.  They are not so we 
-# set them here
```

Comment:
This is fixed by https://github.com/flang-compiler/flang/pull/464/files#diff-af3b638bc2a3e6c650974192a53c7291R30